stripe/src/main/java/com/stripe/android/model/BankStatuses.kt

```diff
@@ -1,11 +1,11 @@
 package com.stripe.android.model
 
-import com.stripe.android.view.FpxBank
+import com.stripe.android.view.Bank
 import kotlinx.parcelize.Parcelize
 import org.jetbrains.annotations.TestOnly
 
 @Parcelize
-internal data class FpxBankStatuses internal constructor(
+data class BankStatuses internal constructor(
```

Comment:
Why is the visibility changed?

Reply:
Ah I misunderstood visibility modifiers, I can keep it `internal` and so can the `AddPaymentMethodListAdapter` class (I think I previously had it public there)